src/manifest.json

```diff
@@ -5,6 +5,15 @@
   "background": {
     "service_worker": "background.bundle.js"
   },
+  "permissions": [
+    "notifications",
+    "identity",
```

Comment:
what is identity here?

Reply:
it's chrome identity api which brings us access_token, every time we need to generate it and also identity api is what generates that initial gmail authorisation popup,